### DISCUSSION TITLE
2024/01/16/juunini-docker-k8s-for-developers/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#185] [Blog Comments]
# 2024/01/16/juunini-docker-k8s-for-developers/

개발자의 본업은 개발이지만, 현실에선 인프라 관리로 인해 많은 고통을 받습니다. 개발에 집중할 수 있도록 도커/쿠버네티스 사용에 필요한 기본 지식을 알려드리겠습

https://tech.cloudmt.co.kr/2024/01/16/juunini-docker-k8s-for-developers/

**Comment by pvvvv** [comment]
'많은 회사들이 인력 부족으로 늘 시간에 쫓겨 고통을 받습니다. 위와 같은 엔지니어링은 순식간에 가능한 것이 아니라, 담당자를 지정하고 지속적으로 관리(구축, 운영, 개선)해야 합니다.' 너무 공감가는 말이네요 ^,^ 잘 읽었습니다
